course/functions.php

```diff
@@ -42,33 +42,24 @@ function course_support() {
 	 * @return void
 	 */
 	function course_scripts() {
-
-		// Register theme stylesheet.
-		wp_register_style(
-			'course-style',
-			get_stylesheet_directory_uri() . '/style.css',
-			array(),
-			wp_get_theme()->get( 'Version' )
-		);
-
-		wp_register_style(
-			'course-sensei-learning-mode',
-			get_stylesheet_directory_uri() . '/learning-mode.css',
-			array(),
-			wp_get_theme()->get(
-				'Version'
-			)
-		);
-
-		// Enqueue theme stylesheet.
+		wp_register_style( 'course-style', get_stylesheet_directory_uri() . '/style.css', array(), wp_get_theme()->get( 'Version' ) );
+		wp_enqueue_script( 'course-header', get_template_directory_uri() . '/assets/js/header.js', [], wp_get_theme()->get( 'Version' ), true );
 		wp_enqueue_style( 'course-style' );
 
-		// TODO: Only Load it if the the page is using learning mode
-		wp_enqueue_style( 'course-sensei-learning-mode' );
+		$has_sensei = function_exists( 'Sensei' );
+
+		/**
+		 * Temporary Hook to skip the learning mode style when the Sensei LMS is able to provide it.
+     * It only use the deprecated style if an old sensei version is installed. 
+		 */
+		$use_style_from_sensei = apply_filters( 'internal_course_theme_use_learning_mode_style_from_sensei', false );
```

Comment:
We should follow our [naming conventions](https://senseip2.wordpress.com/programming-standards-best-practices/naming-conventions/#hooks), so we could call it something like `course_learning_mode_load_styles`.